### PR TITLE
User page layout

### PR DESCRIPTION
**Pull request by PilgrimMemoirs**
@mikelikesbikes @ltw

### CODE REVIEW

File: app/models/availability.rb [hunk 1]
```diff
@@ -28,11 +28,11 @@ class Availability < ActiveRecord::Base
...
-  
+
   def self.today(tz)
     now = Time.now
-    where('start_time between ? and ?', 
-      now.utc.to_s, 
+    where('start_time between ? and ?',
+      now.utc.to_s,
       utc_eod_for_tz(now,tz).to_s)
   end
```
Comment (mikelikesbikes) [inline app/models/availability.rb]: Remove these whitespace/formatting fixes from your PR.

File: app/assets/stylesheets/screen.css.scss [hunk 2]
```diff
@@ -211,10 +219,12 @@ footer {
 	float: left;
 }
 
+
 .right {
-	float: right;
+  float: right;
```
Comment (mikelikesbikes) [inline app/assets/stylesheets/screen.css.scss]: remove this whitespace issue.

File: app/assets/stylesheets/screen.css.scss [hunk 3]
```diff
@@ -152,7 +156,7 @@ footer {
 	padding: 1em 2em;
 	text-align: left;
 	margin-bottom: 1em;
-	clear: both;
+  clear: both;
```
Comment (mikelikesbikes) [inline app/assets/stylesheets/screen.css.scss]: remove this whitespace fix.

**Comment by PilgrimMemoirs** [comment]
Full screen with 2 columns & user info. Screen width > 720px
full-screen_2-columns

Dynamic mobile query to take away floating divs at <720px
mobile_query_no_float

Full screen--dynamically adjusts to absence of user info, appointments
full_screen_no_info

We tried our best, with the help of Matt Dubois, to fix our whitespace errors with git rebase and recommitting hunks we wanted to keep. 

We also added 3 rspec tests to cover the three methods we added to the User model; hopefully test coverage no longer declines.

File: app/models/user.rb [hunk 4]
```diff
@@ -17,6 +17,18 @@ def self.featured_mentors
     limit(10).order("total_kudos DESC")
   end
 
+  def has_bio
```
Comment (ltw) [inline app/models/user.rb]: I would prefer you to use `bio.present?`. `#present?` returns false for empty arrays, empty strings, and falsy values (nil included).

File: app/views/users/show.html.erb [hunk 5]
```diff
@@ -1,43 +1,48 @@
...
+    </div>
+      <%= image_tag @user.gravatar_url, class: 'profpic' %>
+
+    <div class="profile_content_wrap">
+      <p>
+        <a href=<%= href_to_twitters_user(@user) %> id="twitter_logo"><%= image_tag("twitter_black_logo.jpg")%></a>
+        <a href=<%= href_to_twitters_user(@user) %>><%= "@#{@user.try(:twitter_handle)}" %></a>
+      </p>
+
+      <% if @user.has_bio %>
```
Comment (ltw) [inline app/views/users/show.html.erb]: for example
```ruby
if @user.bio.present?
```

**Pull request closed without merging**